lib/member_page.rb

```diff
@@ -0,0 +1,115 @@
+# frozen_string_literal: true
+require_relative 'spanish_congress_page'
+require_relative 'date_of_birth'
+require_relative 'core_ext'
+
+class MemberPage < SpanishCongressPage
+  field :iddiputado do
+    query['idDiputado']
```

Comment:
`query` is a little more ambiguous in this context then when you're already explicitly dealing with a URL. Perhaps something like `query_string` would be a little more obvious?